f5_openstack_agent/client/bigip-onboard-cli.md

```diff
@@ -48,7 +52,7 @@ required params:
 - id: ID of BIG-IP group
 
 positional params:
```

Comment:
add external-physical-mapppings to positional params

Reply:
yes, code has updated